scripts/prettier/prettier-plugin-jsdoc.js

```diff
@@ -0,0 +1,72 @@
+const prettier = require( 'prettier' );
+const parserBabel = require( 'prettier/parser-babel' );
+const parserFlow = require( 'prettier/parser-flow' );
+const { parseComment } = require( '@es-joy/jsdoccomment' );
```

Comment:
`@es-joy/jsdoccomment` and `comment-parser` dependencies are coming from `eslint-plugin-jsdoc` dependency. The correct way would be to install these packages in Sensei, but I didn't want to do it to pollute our dependencies since we'll probably move it to another place (a generic one) in the future.